cmd/get.go

```diff
@@ -36,23 +62,33 @@ This command is useful for creating and updating Release notes in GitHub.
 	RunE: func(command *cobra.Command, args []string) error {
 		fileName := configuration.Config.FileName
 
+		var tmplSrc string
 		var changelog changelog.Changelog
 		var err error
 
 		if printLatest {
 			changelog, err = get.GetLatest(fileName)
 		} else if printVersion != "" {
 			changelog, err = get.GetVersion(fileName, printVersion)
+		} else if outputTemplate == outputNotes {
+			err = fmt.Errorf("notes output only supported with latest or version")
 		} else {
 			changelog, err = get.GetAll(fileName)
 		}
 
+		switch outputTemplate {
+		case outputStandard:
+			tmplSrc = writer.TmplSrcStandard
+		case outputNotes:
+			tmplSrc = writer.TmplSrcNotes
+		}
+
 		if err != nil {
 			return err
 		}
 
 		var buf bytes.Buffer
-		if err := writer.Write(&buf, changelog); err != nil {
+		if err := writer.Write(&buf, tmplSrc, changelog); err != nil {
```

Comment:
`writer.Write` should now expect a string

```suggestion
		if err := writer.Write(&buf, outputType, changelog); err != nil {
```